src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_ForEachStatement.cs

```diff
@@ -852,6 +754,125 @@ private BoundStatement RewriteMultiDimensionalArrayForEachStatement(BoundForEach
             return result;
         }
 
+        /// <summary>
```

Comment:
Nit: keeping this method in its original location would have made the diff easier to follow.

Reply:
It so happened that I started implementing this from scratch, based on what we do for arrays and later noticed that it turned out nearly identical to the one that we had for string. I can move this to where the string one was.

---
In reply to: [153595958](https://github.com/dotnet/roslyn/pull/23363#discussion_r153595958) [](ancestors = 153595958)